Std/Data/List/Basic.lean

```diff
@@ -443,6 +443,22 @@ inductive Sublist {α} : List α → List α → Prop
 
 @[inherit_doc] scoped infixl:50 " <+ " => Sublist
 
+/-- True if the first list is a potentially non-contiguous sub-sequence of the second list. -/
+def isSublist [DecidableEq α] : List α → List α → Bool
+  | [], _ => true
+  | _, [] => false
+  | l₁@(hd₁::tl₁), hd₂::tl₂ =>
+    if hd₁ = hd₂
+    then tl₁.isSublist tl₂
+    else l₁.isSublist tl₂
+
+instance instDecidableIsSublist [DecidableEq α] (l₁ l₂ : List α) : Decidable (l₁.isSublist l₂) :=
+  if h : l₁.isSublist l₂ then
+    isTrue h
+  else
+    isFalse h
```

Comment:
this instance is not necessary, it's a special case of `(b : Bool) : Decidable b`

Reply:
fixed in [f6dd8fb](https://github.com/leanprover/std4/pull/229/commits/f6dd8fbea4cfa82149f044c53ffd8ccaeaebb43e)